gno.land/pkg/integration/testscript_gnoland.go

```diff
@@ -37,7 +37,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const nodeMaxLifespan = time.Second * 30
+const (
+	nodeMaxLifespan = time.Second * 30
+	defaultCoinsStr = "10000000ugnot"
```

Comment:
nit: Use the `std.Coins` declaration instead of a raw string. Even if it's negligible, we should avoid unnecessary parsing. Additionally, it allows us to use Go's numeric notation to declare `defaultCoinsStr` (using `10e8` or `1_000_000_000`). Parsing should only be used for parsing the argument.

Reply:
Both done here: https://github.com/gnolang/gno/pull/4239/commits/9716c12ca744c13f38f60b3f35a85f54383527ef